ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -364,46 +364,63 @@
     </Hds::Form::TextInput::Field>
   {{/if}}
 
-  <Hds::Form::Textarea::Field
-    name='worker_filter'
-    @value={{@model.worker_filter}}
-    @isOptional={{true}}
-    @isInvalid={{@model.errors.worker_filter}}
-    disabled={{form.disabled}}
-    as |F|
-  >
-    <F.Label>
-      {{t 'resources.host-catalog.form.worker_filter.label'}}
-    </F.Label>
-    <F.HelperText>
-      {{t 'resources.host-catalog.form.worker_filter.help'}}
-      <Hds::Link::Inline @href={{doc-url 'worker.create-tags'}}>
-        {{t 'actions.learn-more'}}
-      </Hds::Link::Inline>
-    </F.HelperText>
-    {{#if @model.errors.worker_filter}}
-      <F.Error as |E|>
-        {{#each @model.errors.worker_filter as |error|}}
-          <E.Message>
-            {{error.message}}
-          </E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
-  </Hds::Form::Textarea::Field>
-
-  <Hds::Form::Checkbox::Field
-    name='disable_credential_rotation'
-    checked={{@model.disable_credential_rotation}}
-    disabled={{form.disabled}}
-    {{on 'change' (fn this.toggleDisableCredentialRotation @model)}}
-    as |F|
-  >
-    <F.Label>
-      {{t 'form.disable_credential_rotation.label'}}
-    </F.Label>
-  </Hds::Form::Checkbox::Field>
+  {{#if (feature-flag 'dynamic-credentials-worker-filter')}}
```

Comment:
Can we add a tests to verify this is hidden and displayed when the feature is false and true?